libs/payments/stripe/src/lib/invoice-line-item.factory.ts

```diff
@@ -0,0 +1,35 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { faker } from '@faker-js/faker';
+import { Stripe } from 'stripe';
+
+export const InvoiceLineItemFactory = (
+  override: Partial<Stripe.InvoiceLineItem>
+): Stripe.InvoiceLineItem => ({
+  id: faker.string.alphanumeric(10),
+  object: 'line_item',
+  amount: 500,
+  amount_excluding_tax: 500,
```

Comment:
These should be random integers, probably gen with https://fakerjs.dev/api/number.html#int